### DISCUSSION TITLE
Nested Menu

### DISCUSSION THREAD

**Discussion by uma-uc** [chakra-ui/chakra-ui#6810] [General]
I could not find any docs on nested menu.
Is there a way to do this in chakra-ui?

Thanks in advance

**Comment by fourbian** [comment]
This is the closest I could find:

https://codesandbox.io/s/chakra-nested-sub-menu-8xngmq?file=/src/SubMenu.tsx

**Comment by jaedag** [comment]
Way too late, but you could use another Menu in the original Menu component
